Week-13/url-shortener/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import AboutUs from "./components/AboutUs";
+import Footer from "./components/Footer";
+import Navbar from "./components/Navbar";
+import { Route, Routes } from "react-router-dom";
+import Home from "./pages/Home";
+import Contact from "./components/Contact";
+
+const App = () => {
+  return (
+    <>
+      <Navbar />
```

Comment:
handle highlight the selected page in Navbar
Example : page is about us but home is highlighted
![image](https://github.com/pesto-students/p10-divya-Varu98/assets/11227146/a70bdb24-69b9-4c80-8192-a587a64ee040)